content/_partials/author.html.haml

```diff
@@ -0,0 +1,23 @@
+- authorId = page.author
+- author = site.authors[authorId]
+-if author
+  %div{:class => 'content'}
+    %h3.title
+      About Author
+    %table
+      %tr
+        %td
+          %img{:src => "/images/avatars/#{authorId}.png", :alt => author.name, :height => 128}
```

Comment:
Should check the file existense

Reply:
Agree with Oleg at least partially: make it a jpg.  
I think we should allow larger images but scale them down to a standard size - not sure if 128x128 or 256x256.